mautic/mautic
# [Enhancement] Allow filtering contacts by UTM data within campaigns.

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | 
| New feature? | Yes
| Automated tests included? |
| Related user documentation PR URL | 
| Related developer documentation PR URL | 
| Issues addressed (#s or URLs) | #5868
| BC breaks? |
| Deprecations? | 

#### Description:
See #5868

#### Steps to test this PR:
1. Use the "Contact field value" conditional within a campaign.
2. Set a condition based on UTM Source.
3. Set a UTM source for a contact, and sent that contact through the campaign.
4. The direction of the contact through the campaign should take the filter into account.

![UTM variables shown in contact field value widget](https://i.imgur.com/xoXwYFj.png)
